ideal_weight.rb

```diff
@@ -0,0 +1,7 @@
+p 'Enter your name: '
+name = gets.chomp
+p 'Enter your height: '
+height = gets.chomp.to_i
+
+ideal_weight = (height - 110).positive? ? (height - 110) : 'achieved already'
+puts "#{name.capitalize}, your ideal weight is #{ideal_weight}"
```

Comment:
Make sure you've added an empty string to the end of each file.
https://cifronomika.ru/blog/git-no-newline-at-end-of-file.html